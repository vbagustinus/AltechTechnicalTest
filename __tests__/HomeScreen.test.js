import React from 'react';
import Home from '../src/screens/HomeScreen';
// import renderer from 'react-test-renderer';
// import {render, screen, userEvent} from '@testing-library/react-native';

describe('Home Screen', () => {
  test('Should open HomeScreen', () => {
    // const tree = renderer.create(<Home />).toJSON();
    // expect(tree).toMatchSnapshot();
    // render(<Home />);
  });
});
