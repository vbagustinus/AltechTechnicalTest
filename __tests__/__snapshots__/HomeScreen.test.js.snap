// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Screen Should open HomeScreen 1`] = `null`;
